easy-ui-react/src/components/Placeholder/Placeholder.module.scss

```diff
@@ -0,0 +1,38 @@
+@import "../../styles/common";
```

Comment:
in this file are styles to test our tokens/css/sass build, they aren't meant to represent all established practices

Reply:
got it, makes sense